cli/src/account.rs

```diff
@@ -36,13 +38,18 @@ pub async fn account_prompt(account_handle: AccountHandle) -> Result<(), Error>
 }
 
 // loop on the account prompt
-pub async fn account_prompt_internal(account_handle: AccountHandle) -> Result<bool, Error> {
+pub async fn account_prompt_internal(
+    account_handle: AccountHandle,
+    history: &mut AccountHistory,
+) -> Result<bool, Error> {
     let alias = {
         let account = account_handle.read().await;
         account.alias().clone()
     };
-    let command: String = Input::new().with_prompt(format!("Account \"{alias}\"")).interact()?;
-
+    let command: String = Input::new()
+        .with_prompt(format!("Account \"{}\"", alias))
+        .history_with(history)
+        .interact_text()?;
```

Comment:
So what was the consensus on this then?

Reply:
I thought it caused problems on windows, but I tried a few commands and all worked fine